src/redux/bookStore.js

```diff
@@ -0,0 +1,10 @@
+import { configureStore } from '@reduxjs/toolkit';
+import bookReducer from './books/bookSlice';
+import categoriesReducer from './categories/categoriesSlice';
+
+export default configureStore({
+  reducer: {
+    books: bookReducer,
+    categories: categoriesReducer,
+  },
+});

```

Comment:
You have made a great effort in configuring your redux store. However, your store is not configured correctly.
Please follow the following steps to make sure you have a working redux store.

Kindly install the redux package as well.
```
npm install redux
```

Kindly import the `combineReducers` and `createStore` methods from `redux`.
```
import { combineReducers, createStore } from 'redux'
```

Next, please combine the reducers properly:
```
const reducer = combineReducers({
    books: bookReducer,
    categories: categoriesReducer,
});
```

And finally, configure and export your store:
```
const store = createStore(
  reducer,
);

export default store;
```